Sources/KSCrashRecording/KSCrashReportStore.m

```diff
@@ -146,12 +146,16 @@ - (NSData *)loadCrashReportJSONWithID:(int64_t)reportID
 - (NSArray<NSNumber *> *)reportIDs
 {
     int reportCount = kscrs_getReportCount(&_cConfig);
```

Comment:
returns an int so could be negative. It wouldn't based on the rest of the code, but could, so this calms the sanitizers down.